core/src/main/java/io/substrait/expression/proto/ExpressionProtoConverter.java

```diff
@@ -12,9 +13,11 @@ public class ExpressionProtoConverter implements ExpressionVisitor<Expression, R
       org.slf4j.LoggerFactory.getLogger(ExpressionProtoConverter.class);
 
   private final FunctionLookup lookup;
+  private final RelConverter relConverter;
 
-  public ExpressionProtoConverter(FunctionLookup lookup) {
+  public ExpressionProtoConverter(FunctionLookup lookup, RelConverter relConverter) {
```

Comment:
Do we need to be as specific as RelConverter here or would a narrower interface be sufficient?

Reply:
Changed to use `ExpressionVisitor<Expression, RuntimeException>` here.